### PR TITLE
🔧 Admin Account Data - Delete/Mentor Download

### PR DESCRIPTION
**Pull request by Leundai**
## Description :star2:

- connected download button
- connected delete mentor button

**Note**
Some accounts don't have a verification email inserted so it might fail. 
 
### Downloads
image
image

### Popup confirmation
image

## TODOs :star:
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Review by AngelaLuo49021 — COMMENTED** [review]
@kenetec regarding leo's note about the endpoint failing if there is no verification email do we want to adjust that endpoint?

**Review by AngelaLuo49021 — APPROVED** [review]
Looks good! may have to refactor backend as seen in my previous comment

**Review by AngelaLuo49021 — CHANGES_REQUESTED** [review]
Could you add a verification modal before the account deletes? It should be super simple "Are you sure you want to delete "Angela Luo" No, Yes

**Review by AngelaLuo49021 — APPROVED** [review]
Looks great!

**Comment by kenetec** [comment]
> @kenetec regarding leo's note about the endpoint failing if there is no verification email do we want to adjust that endpoint?

ya i think that's a good idea

File: backend/api/views/download.py [hunk 1]
```diff
@@ -164,6 +164,7 @@ def generate_sheet(sheet_name, row_data, columns):
     try:
         return send_file(
             output,
+            mimetype="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
             attachment_filename="{0}.xlsx".format(sheet_name),
```
Comment (kenetec) [inline backend/api/views/download.py:168]: Can we use `f'abc'` here?

File: frontend/src/utils/api.js [hunk 2]
```diff
@@ -154,10 +154,34 @@ export const fetchMentorsAppointments = () => {
...
+  return instance
+    .get(requestExtension, {
+      responseType: "blob",
+    })
+    .then(
+      (response) => {
+        const url = window.URL.createObjectURL(new Blob([response.data]));
+        const link = document.createElement("a");
+        link.href = url;
+        console.log(response);
```
Comment (kenetec) [inline frontend/src/utils/api.js:166]: delete `console.log`

**Review by kenetec — APPROVED** [review]
awesome job! 😛

**Pull request merged**